File: backend/src/main/java/com/livana/backend/reputation/dto/SbtMintDto.java
package com.livana.backend.reputation.dto;

import java.time.OffsetDateTime;

/**
 * Individual SBT mint record for NGO history view.
 * PRD: "As an NGO operator, I want to see my own SBT history
 * and cumulative reputation, so that I can showcase my track record."
 */
public record SbtMintDto(
    long tokenId,
    String poolAddress,
    long amount,
    String txHash,
    OffsetDateTime blockTimestamp
) {}

File: android/app/src/main/java/com/livana/app/core/network/dto/SbtMintDto.kt
package com.livana.app.core.network.dto

import kotlinx.serialization.Serializable


@Serializable
data class SbtMintDto(
    val tokenId : Long,
    val poolAddress: String,
    val amount : Long,
    val txHash : String,
    val blockTimestamp : String
)
